pkg/utils/types.go

```diff
@@ -14,34 +14,25 @@
 
 package utils
 
-// Defines an immutable type for a webhook. Use NewWebhookType to instantiate.
-type WebhookType struct {
-	isValidating bool
-}
+// WebhookType defines an immutable type for a webhook. Use NewWebhookType to instantiate.
+type WebhookType string
 
-// NewWebhookTypes returns an immutable webhookType.
-func NewWebhookTypes(isValidating bool) WebhookType {
-	return WebhookType{isValidating: isValidating}
-}
+// Valid webhook types.
+const (
+	ValidatingWebhookType WebhookType = "validating"
```

Comment:
Works for me. I initially chose 

```
type WebhookType struct {
 	isValidating bool
 }
```

in case we want to add more properties to the webhook.

Reply:
Sure, but since the webhook types are already defined, and the properties are just constants, I prefer to use methods over fields to reflect the properties.

Reply:
Makes sense to me 👍